components/scream/src/share/field/field_manager.hpp

```diff
@@ -382,7 +382,7 @@ registration_ends ()
   //     we must be able to allocate C bundled.
   //  5) For each cluster, call the function contiguous_superset from scream_utils.hpp
   //     (see that file for details). If the fcn fails to find an ordering of the cluster's
-  //     field that accommodate all bundler request, it will return an empty list.
+  //     field that accommodate all bundled request, it will return an empty list.
```

Comment:
bundled requests

Reply:
Ah, dang. I felt a bit like a "Jeff" for catching these typos, but I clearly can't outperform the master. Good catch!